templates/includes/sidebar.html

```diff
@@ -0,0 +1,9 @@
+<!-- templates/includes/sidebar.html -->
+<div class="sidebar">
+  <ul>
+      <li><a href="#">Home page</a></li>
+      <li><a href="#">Manufacturers</a></li>
+      <li><a href="#">Cars</a></li>
+      <li><a href="#">Drivers</a></li>
```

Comment:
The list items contain anchor tags with dummy links, as specified in the task. This fulfills the requirement for a sidebar with links to different sections.